### PR TITLE
[DEN-226] README update

### PR DESCRIPTION
**Pull request by ssaluja4**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,25 +18,267 @@ go get github.com/EdgeCast/ec-sdk-go
...
+}
+```
+
+### Edge CNAME ###
+
+Our User-Friendly URL, also known as Edge CNAME, takes advantage of an Edge CNAME 
+configuration and a CNAME record to provide a friendlier alternative to a CDN URL. 
+An edge CNAME URL is specific to the platform from which it was configured.
+
+For more information, please read the [official documentation](https://docs.edgecast.com/cdn/index.html#Origin_Server_-_File_Storage/Creating_an_Alias_for_a_CDN_URL.htm)
```
Comment (stevenpaz) [inline README.md:88]: For clarity, perhaps the link text should be "official documentation for Edge CNAMEs"

File: README.md [hunk 2]
```diff
@@ -17,25 +18,267 @@ go get github.com/EdgeCast/ec-sdk-go
...
+	addParams.EdgeCname = cname
+	// ...
+	resp, err := edgecnameService.AddEdgeCname(*addParams)
+// ...
+}
+```
+
+### Customer Origins 
+
+Our Customer Origin Service allows to serve content stored or generated by
```
Comment (stevenpaz) [inline README.md:114]: "allows to serve" - is there a missing word here? Or maybe it should be "allows content to be served"?

File: README.md [hunk 3]
```diff
@@ -17,25 +18,267 @@ go get github.com/EdgeCast/ec-sdk-go
...
+import (
+	"github.com/EdgeCast/ec-sdk-go/edgecast"
+	"github.com/EdgeCast/ec-sdk-go/edgecast/waf"
+)
+// ...
+	policyString := `{
+		// ...
+	}`
+	sdkConfig := edgecast.NewSDKConfig()
+	sdkConfig.IDSCredentials = "IDS CREDENTIALS"
```
Comment (stevenpaz) [inline README.md:249]: The IDS credentials have to be set like this: 
```
idsCredentials := edgecast.IDSCredentials{
		ClientID:     clientID,
		ClientSecret: clientSecret,
		Scope:        scope,
	}
```

File: README.md [hunk 4]
```diff
@@ -44,11 +287,106 @@ import (
...
+	template files used to generate client files and models using swagger api 
+	documentation
+├── Makefile
+        This Makefile should contain all testing and building operations.
+
+```
+
+### Contributing
+
+Please refer to the [contributing.md] (https://github.com/EdgeCast/ec-sdk-go/blob/main/Contributing.md)
```
Comment (stevenpaz) [inline README.md:356]: For some reason, `[contributing.md]` isn't showing up as a link... maybe because of the space between ] and ( ?

**Review by stevenpaz — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (stevenpaz) [reply README.md:88]: Same for the other links to the official docs

File: README.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 # Go SDK for EdgeCast CDN
-This is the official Go SDK for EdgeCast. It contains services that interact with the EdgeCast APIs.
+This is the official Go SDK for EdgeCast. It contains services that interact 
+with the EdgeCast APIs.
```
Comment (defstream) [inline README.md:3]: The official Go SDK for interacting with EdgeCast APIs.

File: README.md [hunk 6]
```diff
@@ -17,25 +18,267 @@ go get github.com/EdgeCast/ec-sdk-go
...
-Simply import the SDK and provide the API credentials provided to you. They may be an API Token or OAuth 2.0 Credentials. Examples are listed below for each feature.
+Simply import the SDK and provide the API credentials provided to you. They may 
+be an API Token or OAuth 2.0 Credentials. Examples are listed below for each 
+feature.
+
+### Customer management ###
+
+Our Customer Management service provides administrative operations to manage 
+Customer accounts and Customer User accounts for a Partner. These operations 
+allows a partner to automate administrative tasks on their customers and
```
Comment (defstream) [inline README.md:29]: allows -> allow

File: README.md [hunk 7]
```diff
@@ -17,25 +18,267 @@ go get github.com/EdgeCast/ec-sdk-go
...
+Our Customer Management service provides administrative operations to manage 
+Customer accounts and Customer User accounts for a Partner. These operations 
+allows a partner to automate administrative tasks on their customers and 
+customer user accounts. 
+
+To use this Customer Management service, use the API Token provided to you.
+
+#### Customer
+
+Customer Account Management Operations allows management of customer accounts
```
Comment (defstream) [inline README.md:36]: allows you to manage customer accounts.

File: README.md [hunk 8]
```diff
@@ -17,25 +18,267 @@ go get github.com/EdgeCast/ec-sdk-go
...
+	addParams.MasterServerGroup = masterServerGroup
+	// ...
+	resp, err := routeDNSService.AddMasterServerGroup(*addParams)
+// ...
+}
+```
+
+### Real Time Log Delivery (RTLD)
+
+Our Real-Time Log Delivery (RTLD) servicee delivers log data in near real-time
```
Comment (defstream) [inline README.md:194]: servicee -> service

File: README.md [hunk 9]
```diff
@@ -44,11 +287,106 @@ import (
...
+- Shikha Saluja: shikha.saluja@edgecast.com
+
+### Testing
+
+To run all test files in the root folder
+```
+go test -v ./...
+```
+Tests should all pass, before and after any work that you do.  If they
+do not, please reach out to the maintainers for help.
```
Comment (defstream) [inline README.md:374]: Tests should all pass before and after any work that you do. If they
do not; please reach out to the maintainers for help.

File: README.md [hunk 10]
```diff
@@ -17,25 +18,267 @@ go get github.com/EdgeCast/ec-sdk-go
 ```
 
 ## Using the SDK
-Simply import the SDK and provide the API credentials provided to you. They may be an API Token or OAuth 2.0 Credentials. Examples are listed below for each feature.
+Simply import the SDK and provide the API credentials provided to you. They may 
+be an API Token or OAuth 2.0 Credentials. Examples are listed below for each 
+feature.
+
+### Customer management ###
```
[the remark this answers is not in the document]
Comment (ssaluja4) [reply README.md]: Updated. thanks.

File: README.md [hunk 11]
```diff
@@ -17,25 +18,267 @@ go get github.com/EdgeCast/ec-sdk-go
...
+
+### Edge CNAME ###
+
+Our User-Friendly URL, also known as Edge CNAME, takes advantage of an Edge CNAME 
+configuration and a CNAME record to provide a friendlier alternative to a CDN URL. 
+An edge CNAME URL is specific to the platform from which it was configured.
+
+For more information, please read the [official documentation](https://docs.edgecast.com/cdn/index.html#Origin_Server_-_File_Storage/Creating_an_Alias_for_a_CDN_URL.htm)
+
+To use the edge CNAME service, use the API Token providede to you.
```
[the remark this answers is not in the document]
Comment (ssaluja4) [reply README.md]: updated.

File: README.md (hunk 1 above)
Comment (ssaluja4) [reply README.md]: updated

File: README.md (hunk 3 above)
Comment (ssaluja4) [reply README.md]: updated

File: README.md (hunk 5 above)
Comment (ssaluja4) [reply README.md]: updated

File: README.md (hunk 6 above)
Comment (ssaluja4) [reply README.md]: updated

File: README.md (hunk 7 above)
Comment (ssaluja4) [reply README.md:35]: updated to "allows you to manage user accounts under a (parent) customer."

File: README.md (hunk 8 above)
Comment (ssaluja4) [reply README.md]: fixed

File: README.md (hunk 9 above)
Comment (ssaluja4) [reply README.md]: updated.

**Review by defstream — APPROVED** [review]

**Review by stevenpaz — APPROVED** [review]

**Pull request merged**